docs/migration_v1.x.md

```diff
@@ -2,33 +2,33 @@
 
 # *Vue i18n Ally* is now **i18n Ally**!
 
-This extension is now come to v1.x stage! From v1.x, we make it not limited to support Vue but other frameworks such as React.js, Angular and more all in one extension!
+This extension has reached the v1.x stage! From v1.x, we are no longer limited to only supporting Vue, but other frameworks such as React.js, Angular and more, all in one extension!
 
 ## Migration from `Vue i18n Ally` (v0.x)
 
-There is not to much works to be done for this migration.
+There is not much to be done for this migration.
 
 1. Install the new `i18n Ally` from [the marketplace](https://marketplace.visualstudio.com/items?itemName=antfu.i18n-ally)
 2. Uninstall the old `Vue i18n Ally`.
 3. You are done. 🎉
 
-Configurations are backward compatible. And all the features for Vue will work as it would do.
+Configurations are backwards compatible. All the features for Vue will work as befores.
```

Comment:
```suggestion
Configurations are backward compatible. All the features for Vue will work as before.
```

Reply:
Yeah, "backward compatible" is more common than "backwards compatible" even though both are correct.

"befores" is me not catching my own errors.